seatunnel-engine/seatunnel-engine-common/src/main/java/org/apache/seatunnel/engine/common/config/YamlSeaTunnelDomConfigProcessor.java

```diff
@@ -357,6 +360,24 @@ private TelemetryMetricConfig parseTelemetryMetricConfig(Node metricNode) {
         return metricConfig;
     }
 
+    private TelemetryLogsConfig parseTelemetryLogsConfig(Node logsNode) {
+        TelemetryLogsConfig logsConfig = new TelemetryLogsConfig();
+        for (Node node : childElements(logsNode)) {
+            String name = cleanNodeName(node);
+            if (ServerConfigOptions.TELEMETRY_LOGS_SCHEDULED_DELETION_ENABLE.key().equals(name)) {
```

Comment:
I think you should do some clean.

Reply:
dont understand what needs to be cleaned up here, a little configuration is needed.